power_system_fault.py

```diff
@@ -73,3 +79,61 @@ def phase_current_b(self):
 
     def phase_current_c(self):
         return self.phase_current_a()
+
+
+class LineToLineFault(Fault):
+    def sequence_current_0(self):
+        return 0j
+
+    def sequence_current_1(self):
+        # we should have this in _init
```

Comment:
I think it's fine to have it here. It's part of the calculation, and we don't need to litter our class with properties like this one. Python will take care of optimizing it anyway. Please remove the comment unless you disagree.